README.md

```diff
@@ -556,8 +556,8 @@ A `git stash` is basically a stack of changes on which you store any changes to
 
 The commands you'll mostly use are `git stash` which places any modifications to the _Working Directory_ on the stash, and `git stash pop` which takes the latest change that was stashed and applies it the to the _Working Directory_ again. 
 
-Just like the stack commands it's named after, `git stash pop` removes the latest stashed change before applying it again. 
-If you want to keep the stashed changes, you can use `git stash apply`, which doesn't remove them from the stash before applying them. 
+Just like the stack commands it's named after, `git stash pop` removes the latest stashed change before applying these changes to the working directory. 
```

Comment:
```suggestion
Just like the [stack commands](https://en.wikipedia.org/wiki/Stack_(abstract_data_type)) it's named after, `git stash pop` removes the latest stashed change before applying these changes to the working directory. 
```

I think a big mistake I make here is explaining this assuming the reader knows how a Stack datastructure works. 
Maybe the link here would help as well.

Reply:
Hey @UnseenWizzard, that's a great idea. This would surely help the readers who might not be familiar with stacks to relate to what is being said.